test/test_system_test_case.rb

```diff
@@ -1,23 +1,25 @@
 require "test_helper"
 
-if ENV["TRAVIS"]
-  require "chromedriver/helper"
-  Chromedriver.set_version "2.35"
-end
-
-class TestSystemTestCase < ActionDispatch::SystemTestCase
-  have_browser = ActionDispatch::SystemTesting.const_defined?(:Browser)
-  if have_browser
-    driver = :selenium
-    driven_by driver, using: :headless_chrome, screen_size: [1920, 1080]
-  else
-    driver = :selenium_chrome_headless
-    driven_by driver, screen_size: [1920, 1080]
+unless ENV["BUNDLE_GEMFILE"].include?('rails5.1-without-puma.gemfile')
```

Comment:
Can we use `const_define?` or `respond_to?` here such as `ActionDispatch::SystemTesting.const_defined?(:Server)` here?
The condition is too fragile. If we rename the gemfile or add more gemfiles for this case, we need to sync to them.

Reply:
Oh...Right.  We can use `ActionDispatch::SystemTesting.const_defined?(:Server)`.